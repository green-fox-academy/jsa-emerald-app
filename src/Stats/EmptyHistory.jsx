import React from 'react';
import { View } from 'react-native';
import { Image } from 'react-native-elements';
import { useSelector } from 'react-redux';
import setThemeStyle from '../Common/theme/setThemeStyle';

const img = require('../../assets/empty_box.png');

export default function EmptyHistory() {
  const { themeMode } = useSelector((state) => state.theme);
  const styles = setThemeStyle(themeMode);

  return (
    <View style={[styles.card, { alignItems: 'center', justifyContent: 'center', paddingVertical: 30 }]}>
      <Image
        source={img}
        style={{ width: 220, height: 220 }}
      />
    </View>
  );
}
