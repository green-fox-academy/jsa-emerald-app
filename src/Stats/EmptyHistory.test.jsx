import React from 'react';
import renderer from 'react-test-renderer';
import EmptyHistory from './EmptyHistory';

describe('EmptyHistory', () => {
  it('render', () => {
    const tree = renderer.create(<EmptyHistory />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
