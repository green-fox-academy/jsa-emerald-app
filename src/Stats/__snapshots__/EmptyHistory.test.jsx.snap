// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyHistory render 1`] = `
<View>
  <Text>
    You haven"t added any transactions yet
  </Text>
</View>
`;
